pkg/cmd/target.go

```diff
@@ -741,3 +760,29 @@ func shootWrapper(targetReader TargetReader, targetWriter TargetWriter, configRe
 	}
 	return nil
 }
+
+//set namespace for current kubectl ctx
+func setKubectlNamespace(kubectlNameSpace string) error {
+
+	if kubectlNameSpace == "" {
+		return errors.New("Namespace must be provided")
+	}
+
+	currentConfig := getKubeConfigOfCurrentTarget()
+
+	out1, err1 := ExecCmdReturnOutput("bash", "-c", "export KUBECONFIG="+currentConfig+"; kubectl config current-context")
```

Comment:
you could use `out` and `err` instead off `out1` and `err1` and `out2` and `err2`